config/config.go

```diff
@@ -74,7 +77,8 @@ type Config struct {
 	OutputFilters      []string
 	SecretStoreFilters []string
 
-	SecretStores map[string]telegraf.SecretStore
+	SecretStores      map[string]telegraf.SecretStore
+	secretStoreSource []secretStoreConfig
```

Comment:
Why not using a `map[string]string` mapping the name to the source?

Reply:
Ok I see, but I would use `map[string][]string` in this case to collect the sources for the same type and print them in groups...

Reply:
Ack. I've made the change.